scripts/exglobal_atmos_analysis.sh

```diff
@@ -748,7 +752,7 @@ cat > gsiparm.anl << EOF
   lobsdiag_forenkf=${lobsdiag_forenkf},
   write_fv3_incr=${write_fv3_increment},
   nhr_anal=${IAUFHRS},
-  ta2tb=${ta2tb},
+  ta2tb=${ta2tb},optconv=${optconv},
```

Comment:
Is this one for turning on the convective clouds?

Reply:
@emilyhcliu This change downweights observations where the humidity Jacobians are large.  @jderber-NOAA found this improved minimisation.

Reply:
I see!  Thanks.